egs/voxceleb/v2/run.sh

```diff
@@ -16,20 +16,30 @@ vaddir=`pwd`/mfcc
 
 
 # The trials file is downloaded by local/make_voxceleb1.pl.
-voxceleb1_trials=data/voxceleb1_test/trials
-voxceleb1_root=/export/corpora/VoxCeleb1
-voxceleb2_root=/export/corpora/VoxCeleb2
+#voxceleb1_trials=data/voxceleb1_test/trials
+voxceleb1_root=/home/data2/VoxCeleb/nivedita/VoxCeleb1
+voxceleb2_root=/home/data2/VoxCeleb/nivedita/VoxCeleb2
 nnet_dir=exp/xvector_nnet_1a
-musan_root=/export/corpora/JHU/musan
+musan_root=/home/data2/VoxCeleb/nivedita/musan
```

Comment:
Would you mind changing this back to the old paths? E.g., 

voxceleb1_root=/export/corpora/VoxCeleb1
voxceleb2_root=/export/corpora/VoxCeleb2
musan_root=/export/corpora/JHU/musan